### PR TITLE
Add reduce option to beroulli_nll

### PR DESCRIPTION
**Pull request by delta2323**
This PR add reduce option to `F.bernoulli_nll` following the design discussed in #2325 with the same spirit of #2519

### CODE REVIEW

File: chainer/functions/loss/vae.py [hunk 1]
```diff
@@ -53,6 +53,11 @@ def bernoulli_nll(x, y):
     where :math:`p = \\sigma(y)`, :math:`\\sigma(\\cdot)` is a sigmoid
     function, and :math:`B(x; p)` is a Bernoulli distribution.
 
+
+    The output is a varialbe whose value depends on the value of
+    the option `reduce`. If it is `'no'`, it holds the elementwise
```
Comment (unnonouno) [inline chainer/functions/loss/vae.py]: Use two backquotes. And check other docstrings.

File: chainer/functions/loss/vae.py [hunk 2]
```diff
@@ -63,15 +68,29 @@ def bernoulli_nll(x, y):
         x (~chainer.Variable): Input variable.
         y (~chainer.Variable): A variable representing the parameter of
             Bernoulli distribution.
+        recude (str): Reduction option. Its value must be either
+            `'sum'` or `'no'`. Otherwise, `ValueError` is raised.
```
Comment (unnonouno) [inline chainer/functions/loss/vae.py]: <pre>:class:`ValueError`</pre> ? Please check HTML generated by sphinx.

File: chainer/functions/loss/vae.py (hunk 1 above)
Comment (delta2323) [inline chainer/functions/loss/vae.py]: Fixed

File: chainer/functions/loss/vae.py (hunk 2 above)
Comment (delta2323) [inline chainer/functions/loss/vae.py]: Fixed

**Comment by unnonouno** [comment]
LGTM! Please rebase or merge the master.

**Comment by unnonouno** [comment]
Check CI result.

**Pull request merged**